src/libs/E2E/utils/NetworkInterceptor.ts

```diff
@@ -155,6 +155,13 @@ export default function installNetworkInterceptor(
         const options = fetchArgsGetRequestInit(args);
         const headers = getFetchRequestHeadersAsObject(options);
         const url = fetchArgsGetUrl(args);
+
+        // Don't process these specific API commands because running them over and over again in the tests hammers the server in a bad way.
```

Comment:
```suggestion
        // Don't process these specific API commands because running them over and over again in the tests increases the size of the notificationPreferences NVP on the server to an infinite size. This is due to the NVP storing this setting once for each user's device, and since the E2E tests use AWS device farm, the user ends up with thousands of different devices, unlike normal users that might only ever have about a dozen. We found the NVP was over 2.5mb in size and that slows down database replication.
```
You might need to break that up into multiple lines, but I think it's a better comment.